onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -2466,7 +2459,14 @@ def _aten_embedding_bag_onnx(
     return result, offset2bag, bag_size, max_indices
 
 
-@torch_op("aten::embedding_bag.padding_idx", trace_only=True)
+@torch_op(
+    (
+        "aten::embedding_bag.padding_idx",
+        "aten::_embedding_bag",
+        "aten::_embedding_bag_forward_only",
+    ),
+    trace_only=True,
+)
```

Comment:
Should we change `padding_idx` to non-Optional value as it's an attribute?

Reply:
Good call. @xiaowuhu do you know what happens when idx is -1?

Reply:
```padding_idx``` must exist in embedding_bag.padding_idx function. 
If it is none, it should be use embedding_bag function.

Reply:
> Should we change `padding_idx` to non-Optional value as it's an attribute?

Tried but couldn't because arguments before it are kwargs. Would this be an issue for the dispatcher?